src/DependencyInjection/Configuration.php

```diff
@@ -203,52 +203,32 @@ public function getConfigTreeBuilder(): TreeBuilder
                                     ->children()
                                         ->arrayNode('fulltext')
                                             ->children()
-                                                ->scalarNode('minimumShouldMatch')
-                                                    ->defaultValue('100%')
```

Comment:
I remove the default value from configuration in order to prevent that symfony add default value for scope that will prevent our fallback mecanism